api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -51,7 +51,8 @@
             + "zookeeper.connect, zookeeper.set.acl, authorizer., super.user"
             + "cruise.control.metrics.topic, cruise.control.metrics.reporter.bootstrap.servers";
 
-    public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "zookeeper.connection.timeout.ms, ssl.cipher.suites, ssl.protocol, ssl.enabled.protocols";
+    public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "zookeeper.connection.timeout.ms, ssl.cipher.suites, ssl.protocol, ssl.enabled.protocols, "
+            + "cruise.control.metrics.topic.num.partitions, cruise.control.metrics.topic.replication.factor";
```

Comment:
I was wondering if we should not allow to do this configuration directly in the `spec.kafka.config` of the `Kafka` resource but exposing these fields in a more user friendly way in the `cruiseControl` field. The problem I see is that if we start in this way, then we could come up to do the same for a tons of cruise control related parameter in Kafka broker (so more related to the metrics reporter) and we fill the `cruiseControl` too much but maybe I am wrong.

Reply:
I do like the idea of keeping the cruise control configurations, including the cruise control metric reporter in one place but I think it may be be better to stay consistent with upstream Cruise Control (at least for now). Diverting from the upstream Cruise Control documentation would add overhead to our code and documentation and there shouldn't be much(if any) tweaking needed on the metric reporter side (avoiding the metric reporter configuration bloat in the Kafka broker configuration!)